### ISSUE TITLE
Missing Morphology display

### ISSUE THREAD

**Issue by TDBuck** [perseids-project/GardenerPrototype#3]
There is no way to display the morphology for individual words in the widget.

**Comment by balmas** [comment]
ok, @TDBuck it's not perfect, but the morphology can now be found in the context menu when you right click on a word in the tree. the position is fixed to the top right of the canvas so that it appears in the same place no matter which word you click. Does this work for you?

**Comment by TDBuck** [comment]
@balmas yeah that works for me. Thanks for getting all this done so quickly.
